index.js

```diff
@@ -0,0 +1,16 @@
+const validateCase = require('./validate-case')
+const validateNum = require('./validate-num')
```

Comment:
You did the process correct Very good job there was lots of room for error this way and it worked.  There is some repeating code in the different files that could have been consolidated.  Don't forget DRY, which also makes me think of work smarter not harder.